src/AutoRest.CSharp/Common/Output/Models/Types/ObjectTypeProperty.cs

```diff
@@ -32,8 +32,8 @@ public ObjectTypeProperty(FieldDeclaration field, InputModelProperty inputModelP
             InitializationValue = field.DefaultValue;
         }
 
-        public ObjectTypeProperty(MemberDeclarationOptions declaration, string parameterDescription, bool isReadOnly, Property? schemaProperty, CSharpType? valueType = null, bool optionalViaNullability = false)
-            : this(declaration, parameterDescription, isReadOnly, schemaProperty, schemaProperty?.IsRequired ?? false, valueType, optionalViaNullability)
+        public ObjectTypeProperty(MemberDeclarationOptions declaration, string parameterDescription, bool isReadOnly, Property? schemaProperty, CSharpType? valueType = null, bool optionalViaNullability = false, SourcePropertySerializationMapping? serializationMapping = null)
+            : this(declaration, parameterDescription, isReadOnly, schemaProperty, schemaProperty?.IsRequired ?? false, valueType: valueType, optionalViaNullability: optionalViaNullability, serializationMapping: serializationMapping)
```

Comment:
It seems the name assignment is not needed here?

Reply:
did not get the point

Reply:
Never mind, I saw there are additional optional properties in between.